code/modules/client/preference_setup/general/01_basic.dm

```diff
@@ -1,74 +1,86 @@
 datum/preferences
-	var/identifying_gender = MALE
+	var/gender = MALE					//gender of character (well duh)
+	var/age = 30						//age of character
+	var/spawnpoint = "Cryogenic Storage" 			//where this character will spawn (0-2).
+	var/metadata = ""
```

Comment:
Metadata was marked as redundant for us and removed.
Returning should be discuss.

Reply:
What should i do with it ? delete it ?